tests/network/bgp/conftest.py

```diff
@@ -0,0 +1,179 @@
+import os
+from collections.abc import Generator
+from pathlib import Path
+from typing import Final
+
+import pytest
+from ocp_resources.config_map import ConfigMap
+from ocp_resources.namespace import Namespace
+from ocp_resources.network_config_openshift_io import Network
+from ocp_resources.node import Node
+from ocp_resources.pod import Pod
+
+from libs.net import netattachdef as libnad
+from libs.net.bgp import (
+    create_cudn_route_advertisements,
+    create_frr_configuration,
+    deploy_external_frr_pod,
+    enable_ra_in_cluster_network_resource,
+    generate_frr_conf,
+)
+from libs.net.node_network import DEFAULT_ROUTE_DEST_V4, BrExNetworkInfo, lookup_br_ex_network_info
+from libs.net.udn import create_udn_namespace
+from tests.network.libs import cluster_user_defined_network as libcudn
+from tests.network.libs import nodenetworkconfigurationpolicy as libnncp
+from tests.network.libs.label_selector import LabelSelector
+from utilities.infra import cache_admin_client, get_node_selector_dict
+
+APP_CUDN_LABEL: Final[dict] = {"app": "cudn"}
+BGP_DATA_PATH: Final[Path] = Path(__file__).resolve().parent / "data" / "frr-config"
+CUDN_BGP_LABEL: Final[dict] = {"cudn-bgp": "blue"}
+CUDN_SUBNET_IPV4: Final[str] = "192.168.10.0/24"
+EXTERNAL_SUBNET_IPV4: Final[str] = "172.100.0.0/16"
+
+
+@pytest.fixture(scope="module")
+def vlan_nncp(vlan_base_iface: str, worker_node1: Node) -> Generator[libnncp.NodeNetworkConfigurationPolicy]:
+    """Creates a NodeNetworkConfigurationPolicy with a VLAN interface on the specified cluster's node."""
+    with libnncp.NodeNetworkConfigurationPolicy(
+        name="test-vlan-nncp",
+        desired_state=libnncp.DesiredState(
+            interfaces=[
+                libnncp.Interface(
+                    name=f"{vlan_base_iface}.{os.environ['VLAN_TAG']}",
```

Comment:
`VLAN_TAG` doesn't exist in the repo yet and is first introduced here, without being set to any value. Is this a new env. var. you are going to set externally to be used here?
Because if this is the case, then you already have the `vlans` pytest argument. If your cluster doesn't support the default range of VLAN tags, which is 1000-1019, then you can and should execute the tests with this parameter, for example `--tc=vlans:963,1005,1271`.
Additionally, how can we set this env. var. when running the tests from jenkins jobs?